src/libprojectM/MilkdropPreset/MotionVectors.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "PerFrameContext.hpp"
+#include "PresetState.hpp"
+
+#include "Renderer/RenderItem.hpp"
+
+/**
+ * @brief Draws a flexible motion vector field.
+ *
+ * This is broken right now, as it only renders a relatively static 1px point grid, with no apparent motion trails.
```

Comment:
is this still true?

Reply:
No, not anymore. It's now properly implemented, I perform it purely on the GPU using a second color attachment and a vertex shader. Some presets like "Radar" and many others now actually look as they should.